springfield/firefox/views.py

```diff
@@ -488,17 +488,15 @@ def get_context_data(self, **kwargs):
     def get_template_names(self):
         variation = self.request.GET.get("variation", None)
         experience = self.request.GET.get("xv", None)
-        locale = l10n_utils.get_locale(self.request)
 
         # ensure variant matches pre-defined value
         if variation not in self.variations:
             variation = None
 
-        if ftl_file_is_active("firefox/download/desktop") and experience != "basic":
-            if locale in ["en-US", "en-CA"]:
-                template = "firefox/download/desktop/download-en-us-ca.html"
-            else:
-                template = "firefox/download/desktop/download.html"
+        if ftl_file_is_active("firefox/download/home") and experience not in ["basic", "legacy"]:
+            template = "firefox/download/home.html"
+        elif ftl_file_is_active("firefox/download/desktop") and experience != "basic":
+            template = "firefox/download/desktop/download.html"
         else:
             template = "firefox/download/basic/base_download.html"
 
```

Comment:
I think there are some views+tests now unused, and angry about their files removed here, so I'd say it's just safe to remove: https://github.com/mozmeao/springfield/blob/01552c14841a3a5db11092f1a5067e86ea1a8a35/springfield/firefox/views.py#L519-L536
and change the relevant https://github.com/mozmeao/springfield/blob/01552c14841a3a5db11092f1a5067e86ea1a8a35/springfield/firefox/tests/test_views.py#L478-L482
to this current view?

I've added that in d2f3293 which looks like nothing important breaks and tests pass.